Assets/HoloToolkit/Input/Tests/KeywordManager.unity

```diff
@@ -0,0 +1,513 @@
+%YAML 1.1
```

Comment:
Please update the README.md on how to use the scene. How to wire up keywords etc.